openslides/utils/autoupdate.py

```diff
@@ -170,31 +31,32 @@ def inform_changed_data(is_delete, *args):
     The other arguments are the changed or deleted Django/OpenSlides model
     instances.
     """
-    if settings.USE_TORNADO_AS_WSGI_SERVER:
-        for instance in args:
-            try:
-                root_instance = instance.get_root_rest_element()
-            except AttributeError:
-                # Instance has no method get_root_rest_element. Just skip it.
-                pass
-            else:
-                access_permissions = root_instance.get_access_permissions()
-                container = {
-                    'dispatch_uid': access_permissions.get_dispatch_uid(),
-                    'collection_string': root_instance.get_collection_string(),
-                    'rest_pk': root_instance.get_rest_pk()}
-                if is_delete and instance == root_instance:
-                    # A root instance is deleted.
-                    container['action'] = 'deleted'
-                else:
-                    # A non root instance is deleted or any instance is just changed.
-                    container['action'] = 'changed'
-                    root_instance.refresh_from_db()
-                    container['full_data'] = access_permissions.get_full_data(root_instance)
-                OpenSlidesSockJSConnection.send_object(json.dumps(container))
-    else:
-        pass
-        # TODO: Implement big variant with Apache or Nginx as WSGI webserver.
+    for instance in args:
+        try:
+            root_instance = instance.get_root_rest_element()
+        except AttributeError:
+            # Instance has no method get_root_rest_element. Just skip it.
+            continue
+
+        access_permissions = root_instance.get_access_permissions()
+        action = 'deleted' if is_delete and instance == root_instance else 'changed'
+        if action == 'changed':
+            root_instance.refresh_from_db()
+            full_data = access_permissions.get_full_data(root_instance)
+
+        for user in get_active_users():
+            channel = Group('user-{}'.format(user.id))
+            output = {
+                'collection': root_instance.get_collection_string(),
+                'id': root_instance.get_rest_pk(),
+                'action': action}
+            if action == 'changed':
+                data = access_permissions.get_restricted_data(full_data, user)
+                if data is None:
+                    # There are no data for the user so he can't see the object. Skip him.
```

Comment:
That is your old code with your old comment. I would like to keep it, because I don't now how the client would react of a change and I don't think we should change it.